tarantool_test.go

```diff
@@ -47,6 +48,8 @@ type Member struct {
 	Val   uint
 }
 
+var contextDoneErrRegexp = regexp.MustCompile(`^context is done \(request ID [0-9]+\)$`)
```

Comment:
I'm a little confused by the use of regular expressions to calculate the error type. regexp is slow compared to errors.Is. Why can't we use the errors.Is construct in this case?

Reply:
We could extend the ClientError to handle it:

https://github.com/tarantool/go-tarantool/blob/0ccdee28735fa907a7934b66678eeb1783b4fdb0/errors.go#L25-L30
https://github.com/tarantool/go-tarantool/blob/0ccdee28735fa907a7934b66678eeb1783b4fdb0/errors.go#L55-L64

Reply:
Since it's tests, there is no performance issue is questioned. So using regexp is ok here.
Returning here a ClientError or something else changes API.
I'm not intended to change the existing API, I'm intended just to make more informative the existing one.